web/src/main/webapp/WEB-INF/config/config-service-search.xml

```diff
@@ -52,7 +56,9 @@
       application search in statistics.
       ]]>
       </documentation>
-      <class name=".services.main.XmlSearch"/>
+      <class name=".services.main.XmlSearch">
+        <param name="maxRecords" value="100" />
```

Comment:
Same with this parameter. Move it to settings somehow, so we can configure easily if we want others to harvest us or not.